go/tools/asthelpergen/asthelpergen_test.go

```diff
@@ -42,8 +43,6 @@ func TestFullGeneration(t *testing.T) {
 		require.Contains(t, contents, "http://www.apache.org/licenses/LICENSE-2.0")
 		applyIdx := strings.Index(contents, "func (a *application) apply(parent, node AST, replacer replacerFunc)")
 		cloneIdx := strings.Index(contents, "CloneAST(in AST) AST")
-		if applyIdx == 0 && cloneIdx == 0 {
-			t.Fatalf("file doesn't contain expected contents")
-		}
+		assert.False(t, applyIdx == 0 && cloneIdx == 0, "file doesn't contain expected contents")
```

Comment:
Seems like a redaction of severity from `Fatalf` to `assert`. Should this not be at least `require`?